index.html

```diff
@@ -66,10 +65,9 @@ <h3>YouTube Video</h3>
             </blockquote>
             <p>&#169; 2020 <a href="mailto:alarseneault@hotmail.com?subject=Message%20from%20CSD120%20Website">Alain Arseneault</a></p>
             <picture>
-                <source srcset="images/sc-logo.svg" style="width: 70px;height: auto;">
-                <source srcset="images/sc-logo.png" media="(max-width:400px)">
-                <source srcset="images/sc-logo@1.5x.png" media="(max-width:800px)">
-                <img src="images/sc-logo@2x.png" alt="images/sc-logo@2x.png">
+                <source srcset="images/sc-logo.svg" type="image/svg+xml">
+                <img src="images/sc-logo.png" alt="Sault College logo" width="70"
+                    srcset="images/sc-logo.png, images/sc-logo@1.5x.png 1.5x, images/sc-logo@2x.png 2x">
```

Comment:
Use `<source>` to provide the alternate format.  For resolution switching you can do this directly in the `<img>` using the srcset attribute